api/src/test/java/org/openmrs/module/ModuleClassLoaderTest.java

```diff
@@ -111,11 +112,63 @@ public void shouldResourceBeIncluded_shouldReturnTrueIfFileMatchesAndModuleVersi
 	}
 	
 	/**
-	 * @verifies return false if file matches and module version does not match
+	 * @throws MalformedURLException
 	 * @see ModuleClassLoader#shouldResourceBeIncluded(Module, java.net.URL, String, java.util.Map)
 	 */
 	@Test
```

Comment:
The two changes above this are not in the original commit.
Some times it may be easier to just manually put in some changes.

Reply:
I did :-)
I just force pushed too early.